src/scenes/main.scene.tsx

```diff
@@ -1,11 +1,11 @@
 import { CanvasPod } from '@/pods/canvas/canvas.pod';
 import { ToolbarPod } from '@/pods/toolbar/toolbar.pod';
-import classes from './main.scene.module.css';
 import {
   CanvasViewSettingsProvider,
   ModalDialogProvider,
 } from '@/core/providers';
 import { ModalDialog } from '@/common/components';
+import classes from './main.scene.module.css';
 
 export const MainScene: React.FC = () => {
```

Comment:
Table header background color is gone ¿should be added some secondary color for the table headers?